src/kudu/consensus/consensus_queue.cc

```diff
@@ -713,10 +724,15 @@ Status PeerMessageQueue::RequestForPeer(const string& uuid,
     vector<ReplicateRefPtr> messages;
     int max_batch_size = FLAGS_consensus_max_batch_size_bytes - request->ByteSize();
 
+    ReadContext read_context;
+    read_context.for_peer_uuid = &uuid;
```

Comment:
nit: curly brace initializer

Reply:
Will retain this for now. tp2 build was emitting warning for using that style - not sure whats wrong.